data/hai/hai reveal 5 philosophers.txt

```diff
@@ -848,19 +945,62 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
+			choice
+				`	"It's good to hear you have the ground combat situation under control."`
+					goto control
+				`	"That equipment sounds ominous."`
+			`	Kopkow gives you a disturbing smile that has an 'I love my work' appearance. He departs as everyone else assembles.`
+				goto leave
+			label control
+			`	"It's more of a collaboration, really, but like I said: need-to-know basis." He nods to you and departs as everyone else assembles.`
+			label leave
+			`	After a large meeting about logistics, and an unsurmountable buffet, everyone disperses. General Choot'k finds you to discuss critical tactical matters.`
+			`	"There's a problem with the plan," he says, "which we found after more analysis of the scans. The bay where we need to land already has ships inside, and if we try to fly in there, they'll blow us to pieces. You have lure them out and kill them all."`
+			choice
+				`	"How do I lure them out?"`
+					goto how
+				`	"Can't we blow them up on the way down?"`
+			`	"Have you ever tried to land on a pile of blown-up space ships in the wreck of a spaceport a half kilometer underground? It ain't easy.`
+			label how
+			`	"I've spent a lot of time interrogating pirate leaders these past few weeks, and they all have one thing in common: narcissism. Threaten their precious egos, and they'll fight when they should hide. All you have to do is demand tribute from their planet and the fools will rush out and fight you.`
+			`	"The pirates won't believe it if we demand the tribute, but they will believe it if you demand it. Then their defense fleets will rush out. You have to defeat them on your own to maintain the ruse. Eventually they'll give up and pay tribute. That's when we know it's safe to land."`
+			choice
+				`	"Sounds good. I look forward to the tribute payments."`
+					goto done
+				`	"Why do I have to do that myself?"`
+			`	"In your culture, governments don't demand tribute from enemy planets." He shrugs. "The pirates will see it as an obvious trap if we do it. You, though? They'll see a pilot taking advantage of the situation. Just lure them out, and blow them up, while we hold off any incoming ships."`
+			choice
+				`	"Sounds good. I look forward to the tribute payments."`
+					goto done
+				`	"Why won't our fleet help me destroy the pirates coming from the planet?"`
+			`	"Ah, they could, but they won't. Consider it a test. I want to see you prove yourself in battle before I open the entrance to <planet>."`
+			label done
+			`	He seems to admire your ship for a moment. "If you succeed today you shall be Hai-friend, wherever you go."`
```

Comment:
I'm glad this part flows so well now. It makes so much more sense. Before, Choot'k just sounded like he was just being annoying by making you fight the tribute fleet yourself (it is still annoying...)

Reply:
Not that much now that it's been balanced, I hope?

Reply:
Well, I was able to defeat it this time. But I hope you didn't read what I wrote last night...

Reply:
He isn't "being annoying." You are just suffering from culture shock. If the challenge were unsurmountable, he would not have put it before you (after a balance PR).

Reply:
I know he isn't meant to be annoying, I'm happier that it makes sense now.